styles_src/default.scss

```diff
@@ -50,7 +54,7 @@ $textContainerMarginTop: 22px;
   }
```

Comment:
add a negative `margin-right: -$defaultMargin;` to the legend container. This way the single columns (no matter if it's one or two) will be aligned to the right side.